greek/use_the_rope/grab_the_ax/grab_the_ax

```diff
@@ -0,0 +1,4 @@
+Αρπάζεις το τσεκούρι το οποίο βρίσκεται κάτω απο το παράθυρο σε θήκη για περίπτωση φωτιάς και κόβεις το σχοινί.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.